public/Day 60/index.html

```
@@ -1,17 +1,28 @@
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Ultimate Travel Planner | Trip Intelligence">
+    <meta property="og:description"
+        content="Plan flights, hotels, and activities in one place with our intuitive travel dashboard.">
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:title" content="Ultimate Travel Planner | Trip Intelligence">
+    <meta name="twitter:description"
+        content="Plan flights, hotels, and activities in one place with our intuitive travel dashboard.">
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ link ] must be self closed.

(tag-self-close)